src/test/java/com/example/ajaxonboardingproject/UserRepositoryTests.kt

```diff
@@ -0,0 +1,122 @@
+package com.example.ajaxonboardingproject
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import com.example.ajaxonboardingproject.model.CinemaHall
+import com.example.ajaxonboardingproject.model.Movie
+import com.example.ajaxonboardingproject.model.MovieSession
+import com.example.ajaxonboardingproject.model.Role
+import com.example.ajaxonboardingproject.model.ShoppingCart
+import com.example.ajaxonboardingproject.model.Ticket
+import com.example.ajaxonboardingproject.model.User
+import com.example.ajaxonboardingproject.repository.UserRepository
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
+import java.time.LocalDateTime
+
+@SpringBootTest
+class UserRepositoryTests {
+    @Autowired
+    private lateinit var userRepository: UserRepository
+
+    @BeforeEach
+    fun clearCollection() {
+        userRepository.deleteAll().subscribe()
```

Comment:
it's better to block here, because it will delete records asynchronously, that could lead to flaky tests
```suggestion
        userRepository.deleteAll().block()
```

Reply:
Done!